eslint-server/src/server.ts

```diff
@@ -343,7 +345,35 @@ documents.onDidOpen((event) => {
 	if (!supportedLanguages[event.document.languageId]) {
 		return;
 	}
-	const style = settings.standard.semistandard ? 'semistandard' : 'standard';
+	const usePackageJson = settings.standard.usePackageJson;
+	let pkgStyle
+	let pkgOptions
+	if (usePackageJson) {
+		const pkgPath = path.join(workspaceRoot, 'package.json');

```

Comment:
When there's no `package.json` file presented, what would you suggest doing with the project? Lint it or not?